net/unbound/files/unbound.sh

```diff
@@ -1067,28 +1065,33 @@ unbound_start() {
 
     unbound_control
   fi
+
```

Comment:
okay then, I would suggest a slight change in course. Instead of _ismain() lets have _resolv_setup(). Called last, performs the checks, and writes the resolve file. Add one echo statement:
`echo "# /tmp/resolve.conf generated by Unbound UCI $( date )"`